### ISSUE TITLE
add program passed as string to dis module.

### ISSUE THREAD

**Issue by f0f7edab-0f77-471a-8c8d-7bde907388b2** [python/cpython#88571]
BPO | [44405](https://bugs.python.org/issue44405)
--- | :---
Nosy | @terryjreedy, @ncoghlan, @stevendaprano, @serhiy-storchaka, @1st1, @eamanu, @CCLDArjun
PRs | <li>python/cpython#26714</li>
Files | <li>[dis.patch](https://bugs.python.org/file50107/dis.patch "Uploaded as text/plain at 2021-06-13.00:44:23 by @CCLDArjun")</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2021-06-13.00:44:23.977>
labels = ['type-feature', 'library', '3.11']
title = 'add program passed as string to dis module.'
updated_at = <Date 2022-03-18.15:17:19.473>
user = 'https://github.com/CCLDArjun'
```

bugs.python.org fields:
```python
activity = <Date 2022-03-18.15:17:19.473>
actor = 'serhiy.storchaka'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = ['Library (Lib)']
creation = <Date 2021-06-13.00:44:23.977>
creator = 'CCLDArjun'
dependencies = []
files = ['50107']
hgrepos = []
issue_num = 44405
keywords = ['patch']
message_count = 9.0
messages = ['395720', '395722', '395723', '395724', '395725', '395727', '395731', '398046', '415505']
nosy_count = 7.0
nosy_names = ['terry.reedy', 'ncoghlan', 'steven.daprano', 'serhiy.storchaka', 'yselivanov', 'eamanu', 'CCLDArjun']
pr_nums = ['26714']
priority = 'normal'
resolution = None
stage = 'patch review'
status = 'open'
superseder = None
type = 'enhancement'
url = 'https://bugs.python.org/issue44405'
versions = ['Python 3.11']
```

</p></details>

**Comment by hugovk** [comment]
New issue for documenting the CLI: https://github.com/python/cpython/issues/108826

And PR: https://github.com/python/cpython/pull/108827

**Comment by serhiy-storchaka** [comment]
I suggest to close this issue.

**Comment by picnixz** [comment]
We documented the `dis` module, but the question on whether we use `dis` with a `-c` argument remains. I find it actually helpful to do something like `python -m dis -c "return 1"` just to check some simple constructions and the output but apart from that, programs would be multiline and wouldn't benefit from this.

If we want to support inline arguments for dis, we should also support it for tokenize, ast and symtable in one go and this should warrant a DPO thread: https://discuss.python.org/c/ideas/6.

Closing as `not planned` for now.
